pkg/service/HelmAppService.go

```diff
@@ -545,14 +545,28 @@ func (impl HelmAppServiceImpl) installRelease(request *client.InstallReleaseRequ
 // 3. expose this method over grpc
 // 4. write rest handler, router, and servie in orchestrator
 // 5 .invoke this method using grpc
-func (impl HelmAppServiceImpl) GetNotes(ctx context.Context, installReleaseRequest *client.InstallReleaseRequest) (string, error) {
-
-	release, err := impl.installRelease(installReleaseRequest, true)
+func (impl HelmAppServiceImpl) GetNotes(ctx context.Context, request *client.InstallReleaseRequest) (string, error) {
+	releaseIdentifier := request.ReleaseIdentifier
+	helmClientObj, err := impl.getHelmClient(releaseIdentifier.ClusterConfig, releaseIdentifier.ReleaseNamespace)
+	chartSpec := &helmClient.ChartSpec{
+		ReleaseName:   releaseIdentifier.ReleaseName,
+		Namespace:     releaseIdentifier.ReleaseNamespace,
+		ChartName:     request.ChartName,
+		CleanupOnFail: true, // allow deletion of new resources created in this rollback when rollback fails
+		MaxHistory:    0,    // limit the maximum number of revisions saved per release. Use 0 for no limit (default 10)
+		RepoURL:       request.ChartRepository.Url,
+	}
+	HelmTemplateOptions := &helmClient.HelmTemplateOptions{}
+	release, err := helmClientObj.GetNotes(chartSpec, HelmTemplateOptions)
 	if err != nil {
 		impl.logger.Errorw("Error in fetching Notes ", "err", err)
 		return "", err
 	}
-	return release.Info.Notes, nil
+	if release == nil {
+		return "", errors.New("release is found nil")
```

Comment:
add logger, put request there, change error message in response "no release found with the name = release name"

Reply:
done